Lib/defcon/objects/contour.py

```diff
@@ -268,8 +280,7 @@ def reverse(self):
         This will post *Contour.WindingDirectionChanged*,
         *Contour.PointsChanged* and *Contour.Changed* notifications.
         """
-        # No ufoLib analogue to ReverseContourPointPen
-        from robofab.pens.reverseContourPointPen import ReverseContourPointPen
+        from defcon.pens.reverseContourPointPen import ReverseContourPointPen
```

Comment:
Hold off on moving it to ufoLib for now. We still need to nail down what we're going to do about pens in general and this one is sufficiently reusable enough to be part of the overall pens package.